Dockerfile

```
@@ -36,6 +38,9 @@ COPY ./deploy/gunicorn/edumfaapp.py /opt/edumfa/app.py
 # Create directory for user scripts
 RUN mkdir -p /opt/edumfa/user-scripts
 
+# Update this path when updating the Python version!
+RUN ln -s /usr/local/lib/python3.14/site-packages/edumfa/ /opt/edumfa/edumfa-package
```

```suggestion
RUN ln -s $(python3 -c "import site; print(site.getsitepackages()[0])")/edumfa/ /opt/edumfa/edumfa-package
```
Think that should solve the version dependence.

---

That's a good idea! I've also added an assertion to avoid us silently breaking it. Not the prettiest though..

---

Do we really want to link the complete directory or only the static files?

---

Is there an advantage to only linking the static dir?
If we link the complete directory users could use this for easy monkey-patching. Not saying we'd support or mention it, but it might come in handy for someone :smile: .

---

agreed during meeting to leave as-is